homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,32 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int Fn_2{0};
+    int Fn_1{1};
+
+    if(sequence==0){
+        return 0;
+    }else if(sequence==1){
+        return 1;
+    }else{
+        
+        for(int i=0;i<sequence;i++)
+        {
+            Fn_1+=Fn_2;
+            Fn_2=Fn_1-Fn_2;           
+        }
+        return Fn_2;
+    }
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence==0){
+        return 0;
+    }else if(sequence==1){
+        return 1;
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`